app/models/instrument.rb

```diff
@@ -14,6 +14,13 @@
 
 class Instrument < ActiveRecord::Base
 
+  validates :name, presence: true
+  validates :years_played, presence: true
+  validates :experience_level, presence: true
+  validates :is_primary, presence: true
```

Comment:
heh this is a boolean so i think you need to do the check that I did for is_searching